# HG changeset patch
# User Rafael Laboissiere <rafael@debian.org>
# Date 1238575378 -7200
#      Wed Apr 01 10:42:58 2009 +0200
# Branch release-3-0-x
# Node ID af62948f711b0d2b604cd860e2dc7f0dd6d73119
# Parent  278afaecddd48b39c1b639c32113db3d7df8c85c
remove strange volatile declaration

diff --git a/liboctave/EIG.cc b/liboctave/EIG.cc
--- a/liboctave/EIG.cc
+++ b/liboctave/EIG.cc
@@ -94,17 +94,17 @@ EIG::init (const Matrix& a, bool calc_ev
   double *tmp_data = atmp.fortran_vec ();
 
   Array<double> wr (n);
   double *pwr = wr.fortran_vec ();
 
   Array<double> wi (n);
   double *pwi = wi.fortran_vec ();
 
-  volatile octave_idx_type nvr = calc_ev ? n : 0;
+  octave_idx_type nvr = calc_ev ? n : 0;
   Matrix vr (nvr, nvr);
   double *pvr = vr.fortran_vec ();
 
   octave_idx_type lwork = -1;
   double dummy_work;
 
   double *dummy = 0;
   octave_idx_type idummy = 1;
